lib/MetaCPAN/Script/Mapping.pm

```diff
@@ -145,14 +161,27 @@ sub run {
             }
         }
 
+        if ( $self->arg_verify_mapping ) {
+            $self->check_health;
+            unless (
```

Comment:
`unless` being **evil** is quite controversial in the _Perl_ community.
[is `if` better than `unless`?](https://stackoverflow.com/questions/3048726/usage-of-if-versus-unless-for-perl-conditionals)

but I agree with this statement:
["_write code that is clearer and more expressive_"](https://stackoverflow.com/a/3049257/7774231)
> For simple negations I find it clearer than a **negated** ` if ` -- it's **easy to miss** that leading ` ! ` when reading code.

So, I critically reviewed the usage of `unless` and could not find that it would gain any readability changing it.

Reply:
Please notice that it is also **not strictly avoided** in the project:
https://github.com/metacpan/metacpan-api/blob/10a6d7d1d3f2cec635d68cea534f62ce790b5f59/lib/MetaCPAN/Script/Mapping.pm#L198
```
    die "update_index requires patch_mapping\n"
        unless $self->patch_mapping;
```